### PR TITLE
[FIX] Fixes broken triggers.

### PR DESCRIPTION
**Pull request by cauefcr**
Fixed conditions so that triggers work again.

### CODE REVIEW

File: src/lib/triggers.js [hunk 1]
```diff
@@ -172,9 +168,22 @@ class Triggers {
 							this.fire(trigger);
 						}, parseInt(condition.value, 10) * 1000);
 						break;
+					case 'chat-opened-by-visitor':
+						const openFunc = () => {
+							const { user } = store.state;
+							if (user) {
+								if (trigger.runOnce) { store.off('change', openFunc); }
+								setTimeout(() => {
+									this.fire(trigger).then(console.log).catch(console.error).finally(console.debug);
```
Comment (KevLehman) [inline src/lib/triggers.js:177]: 🤔

File: src/routes/TriggerMessage/component.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ export default class TriggerMessage extends Component {
 	componentDidUpdate() {
 		let height = 0;
 
-		this.ref.current.base.children.forEach((el) => {
+		[...this.ref.current.base.children].forEach((el) => {
```
Comment (KevLehman) [inline src/routes/TriggerMessage/component.js:23]: Do you need the spread here? 👀

File: src/lib/triggers.js (hunk 1 above)
Comment (tiagoevanp) [reply src/lib/triggers.js:177]: console.log is necesseary?

File: src/routes/TriggerMessage/component.js (hunk 2 above)
Comment (cauefcr) [reply src/routes/TriggerMessage/component.js:23]: yes, forEach is undefined without it for some reason

File: src/lib/triggers.js (hunk 1 above)
Comment (cauefcr) [reply src/lib/triggers.js:177]: console.log is forgotten code, setTimeout was unfortunately needed for it to work

Comment (cauefcr) [reply src/lib/triggers.js]: forget that, it's useless

File: src/routes/TriggerMessage/component.js (hunk 2 above)
Comment (tiagoevanp) [reply src/routes/TriggerMessage/component.js:23]: so... children is not an iterable array type indeed...

Comment (tiagoevanp) [reply src/routes/TriggerMessage/component.js]: @cauefcr is doing almost the same as Array.from(childrenEls), that's enough to me...

Comment (KevLehman) [reply src/routes/TriggerMessage/component.js]: Yep, the thing is that ... and .foreach should behave similar for arraylike elements (like children lists) 👀 that's why i was wondering if it was needed haha

**Review by tiagoevanp — APPROVED** [review]

**Pull request merged**